sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p><code>!pip install --force-reinstall sagemaker==2.70.0</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I can make this change but any reason we want to hardcode the version instead of using the latest one ?

Reply:
Depending on SageMaker PySDK versions, the notebooks might break. We want to avoid any such problems. We'd better fix the versions that we have tested the notebooks with.

Reply:
<p>Remove all outputs</p>